File: C++ - Class 8/simpleInterest.cpp
//VS03
#include <iostream.h>

int main()
{
    float p,r,t,i;
    cout<<"Enter the principal for calculating simple interest."<<endl;
    cin>>p;
    cout<<"Enter the rate of interest."<<endl;
    cin>>r;
    cout<<"Enter number of years for interest to collect."<<endl;
    cin>>t;
    i=(p*r*t)/100;
    cout<<"The simple interest for "<<p<<" over "<<t<<" years at "<<r<<"% interest is "<<i<<endl;
    return 0;
}

File: BlueJ Java - Class 9 and 10/ClassIX/simpleInterest.java
import java.io.*;
import java.util.*;
/**
 * A class to calculate simple interest.
 *
 * @author (Vinaayak G Dasika)
 * @version (28/7/2020)
 */
public class simpleInterest
{
    double p,n,r,i;
    void main()
    {
        Scanner S = new Scanner(System.in);
        System.out.println("Enter the principal");
        p = S.nextDouble();
        System.out.println("Enter the number of years");
        n = S.nextDouble();        
        System.out.println("Enter the rate of interest");
        r = S.nextDouble();
        i = (p*n*r)/100.00;
        System.out.println("The interest is: "+i+".");
    }
}
